src/mscorlib/src/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -62,5 +62,12 @@ public struct KeyValuePair<TKey, TValue> {
             s.Append(']');
             return StringBuilderCache.GetStringAndRelease(s);
         }
+
+        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Yep I called that out in https://github.com/dotnet/corefx/issues/13746#issuecomment-265829858 as well. I'm fine with it but I wanted to make sure others like @karelz are fine with it not being hidden on desktop.